pylint/checkers/__init__.py

```diff
@@ -123,8 +123,8 @@ def table_lines_from_stats(
                 ("error", "NC"),
             ]
 
-    for index, _ in enumerate(new):
-        new_value = new[index][1]
+    for index, value in enumerate(new):
```

Comment:
@timmartin Could you maybe extract these changes unrelated to adding the checker itself (but actually fixing issues) in a separate PR? We're likely to squash merge this PR (we generally do so) so the commit message for this change would be a bit misleading.

Reply:
Done. I'll leave these changes in this branch for now, or it will fail in CI. Will this get rebased before the merge so that we can eliminate these lines from the squashed commit?

Reply:
Depends on the timeline of this PR I think. Pierre scheduled it for `2.14` since we really want to devote all our attention to getting `2.13` asap, but I had some spare time and went ahead and reviewed already. Depending on the time when we really get to reviewing this and the state of `main` we might need to rebase or merge `main` itself.

Reply:
I merged the changes so you could merge `main` to eliminate the lines from this PR/commits.

Reply:
Done. I did a squash while I was at it.

Reply:
Tests are all passing now. Let me know if there are any other things for me to do here.